drivers/spi/spi_context.h

```diff
@@ -112,13 +112,27 @@ static inline void spi_context_release(struct spi_context *ctx, int status)
 static inline int spi_context_wait_for_completion(struct spi_context *ctx)
 {
 	int status = 0;
+	uint64_t ms;
```

Comment:
do we really need to store the result in `uint64_t`? Sounds like the results should fit in `uint32_t`, meaning clamping to `UINT_MAX` could even be avoided.